### PR TITLE
Support db migration service

### LINKED ISSUE #388: Azure Database Migration Service - PQ3
Resource support for Azure Database Migration Service

### PR DESCRIPTION
**Pull request by sathish-progress**
### Description

Support DB migration Service(s)

### Issues Resolved

#388 

### Check List

- [X] New functionality includes tests
- [X] All tests pass
- [X] `rake lint` passes
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: test/integration/verify/controls/azure_db_migration_services.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+service_name = attribute(:inspec_db_migration_service_name, default: nil)
+sku_name = attribute(:inspec_db_migration_service_sku_name, default: nil)
+location = attribute(:location, default: nil)
+
+control 'azure_db_migration_services' do
+  describe azure_db_migration_services do
```
Comment (soumyo13) [inline test/integration/verify/controls/azure_db_migration_services.rb:6]: After the control block, we should add the impact and the description. This is how we do in was. Do we have to do the same?

File: test/integration/verify/controls/azure_db_migration_service.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+resource_group_name = attribute(:resource_group, default: nil)
+service_name = attribute(:inspec_db_migration_service_name, default: nil)
+sku_name = attribute(:inspec_db_migration_service_sku_name, default: nil)
+location = attribute(:location, default: nil)
+
+control 'azure_db_migration_service' do
+  describe azure_db_migration_service(resource_group: resource_group_name, service_name: service_name) do
```
Comment (soumyo13) [inline test/integration/verify/controls/azure_db_migration_service.rb:7]: After the control block, we should add the impact and the description. This is how we do in was. Do we have to do the same?

**Comment by soumyo13** [comment]
Left out some comments. Please check once. LGTM.

**Review by soumyo13 — APPROVED** [review]

File: test/integration/verify/controls/azure_db_migration_services.rb (hunk 1 above)
Comment (sathish-progress) [reply test/integration/verify/controls/azure_db_migration_services.rb:6]: it is not mandatory ..

File: test/integration/verify/controls/azure_db_migration_service.rb (hunk 2 above)
Comment (sathish-progress) [reply test/integration/verify/controls/azure_db_migration_service.rb:7]: it is not mandatory ..

**Pull request merged**